.github/workflows/APIDigesterCheck.yml

```diff
@@ -0,0 +1,42 @@
+name: Swift API Digester Functionality Check
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # This will run the action every night at midnight
```

Comment:
Since this workflow will post a Slack message in case of failure, can we run it instead every day at 8.00 or something like that?